tools/tools.go

```diff
@@ -0,0 +1,28 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build tools
+// +build tools
+
+package tools
+
+import (
+	// mage notice will fail without this, since it'll try and fetch this with `go install`
+	_ "go.elastic.co/go-licence-detector"
+
```

Comment:
```suggestion

	_ "github.com/elastic/go-licenser"
```

I think you'll need this as well to automatically add license headers